### PR TITLE
Improve testing suite

### PR DESCRIPTION
**Pull request by NeffIsBack**
What have been done:

- Removed `bh__owned.py` module as it is redundant now and throws errors.
- Added creds for testing in the GOAD lab
- Removed tests with `--options` as it technically doesn't add any testing value (just printing a string)
- Solved the "data/test_uernames.txt" testing issue with setting the `cwd` from the opened subprocess
- Some QOL changes like having a time tracking feature when tests are finished
- `--print-failures` defaults to true now as i find that very very useful
- The tests are also tracking tracebacks now:
image

### CODE REVIEW

**Review by mpgn — APPROVED** [review]

**Review by Marshall-Hallenbeck — APPROVED** [review]
I added one little bugfix (it was from the mssql output from my previous changes anyway.

LGTM

**Review by Marshall-Hallenbeck — DISMISSED** [review]
@NeffIsBack if my changes look good feel free to merge.

**Comment by NeffIsBack** [comment]
> @NeffIsBack if my changes look good feel free to merge.

Gonna wait until #225 is merged so we don't have merge conflicts again

**Review by Marshall-Hallenbeck — APPROVED** [review]

**Review by mpgn — APPROVED** [review]

**Pull request merged**
